include/public/acrn_common.h

```diff
@@ -152,7 +152,7 @@ struct acrn_create_vm {
 	uint8_t	 GUID[16];
 
 	/* VM flag bits from Guest OS, now used
```

Comment:
maybe a little bit confusion:
VM flag bits from Guest OS -> VM flag bits *for* Guest OS?

i believe This VM flag is sent/configured from SOS, but for guest OS ,, right?

Reply:
yes it is for guest OS, if modify, because it is a common header file, need to modify dm/hv/sos,
for now can not change it?

Reply:
it's ok, i see.

Reply:
@bzhu5 Would you approve the PR?

Reply:
@lijinxia it is already approved